src/datagrid/DataGrid.tsx

```diff
@@ -927,19 +927,17 @@ export class DataGrid extends React.PureComponent<DataGridProps, DataGridState>
         return <HideShowColumns columns={allColumns} updateColumns={this.updateColumns} />;
     }
 
-    private buildFooterContent():React.ReactElement{
+    private buildFooterContent(): React.ReactElement {
         const {footer} = this.props;
         const {allRows, itemText} = this.state;
         const footerDescription = allRows.length.toString() + " " + itemText;
         let content;
 
-        if (footer !== undefined){
-            content =  footer.footerData !== undefined ?  footer.footerData : footerDescription;
+        if (footer !== undefined) {
+            content = footer.footerData !== undefined ? footer.footerData : footerDescription;
```

Comment:
Do you expect footerData can have 0,'' (empty)values and is it correct?
If not we can replace that one on `content = footer.footerData || footerDescription`

Reply:
@alexander-rassanov 'footerData' is an optional prop so it can be undefined.